js/dist/jed/jed.js

```diff
@@ -0,0 +1,1022 @@
+/**
```

Comment:
@atimmer would it be possible to only have a minified version of this in the plugin?